enterprise/api/subscription.go

```diff
@@ -10,11 +10,10 @@ type SubscriptionPatch struct {
 
 // Subscription is the API message for subscription.
 type Subscription struct {
-	InstanceCount int          `jsonapi:"attr,instanceCount"`
-	ExpiresTs     int64        `jsonapi:"attr,expiresTs"`
-	StartedTs     int64        `jsonapi:"attr,startedTs"`
-	Plan          api.PlanType `jsonapi:"attr,plan"`
-	Trialing      bool         `jsonapi:"attr,trialing"`
-	OrgID         string       `jsonapi:"attr,orgId"`
-	OrgName       string       `jsonapi:"attr,orgName"`
+	ExpiresTs int64        `jsonapi:"attr,expiresTs"`
```

Comment:
I think the subscription should still contain the InstanceCount.

We only intend to remove the instanceCount limit from FREE/TEAM plan, but we don't intend to remove the support to charge user based on instanceCount for the Enterprise plan.